hedyorg/hedy
# 🪲 Fix front-end print statement missing warning for music

This PR solves the issue of warning messages appearing on music adventures when no print statement is present

In spite of not requiring print statements to output, until now submitting a piece of code to a music adventure without including a print statement would result in a warning such as in the following example:

![Warning appears in music](https://github.com/hedyorg/hedy/assets/75997357/33c07cd9-65aa-4b4b-9f07-b39e078f0d3a)


I have solved the issue by checking whether the adventure has any music before issuing the warning in a similar manner to what is done with adventures including turtles


Fixes [#5000](https://github.com/hedyorg/hedy/issues/5000)

**How to test**

Follow these steps to verify this PR works as intended:
* Go to [http://localhost:8080/hedy/1#music](http://localhost:8080/hedy/1#music)
* Type:
```
play C4
```
*Check that no warning appears:

![Warning no longer appears in music](https://github.com/hedyorg/hedy/assets/75997357/7b25bebe-0f52-45c8-b9fb-08e5c8615e4d)
